arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -292,9 +196,9 @@
     "/{resourceUri}/providers/microsoft.insights/metrics": {
       "get": {
         "tags": [
-          "Metric"
+          "Metrics"
         ],
-        "operationId": "Metric_Get",
+        "operationId": "Metrics_Get",
```

Comment:
It should be Metrics_List as you are returning an IEnumerable in the generated code